### PR TITLE
fixed link between timetable and timetable detail

### LINKED ISSUE #286: Timetable click on iOS fails to send data to detail screen
Instead of the selected item being sent, a generated fake is sent. This should be fixed

### PR DESCRIPTION
**Pull request by charles-b-stb**
## Issue
- close #286

## Overview (Required)
- Added in a way to send the selected item to the timetable detail screen

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

## Movie (Optional)
Before | After
:--: | :--:
<video src="https://github.com/user-attachments/assets/41b6fece-c252-498c-bef4-e485f1c08ffc" width="300"  alt="before" title="Before">  |  <video src="https://github.com/user-attachments/assets/c5f5fd2e-38a4-43ba-aa54-6f1d964abdcb" width="300"  alt="after" title="After">

### CODE REVIEW

File: app-ios/Sources/TimetableFeature/TimetableListView.swift [hunk 1]
```diff
@@ -91,10 +91,10 @@ struct TimetableListView: View {
             LazyVStack {
                 ForEach(store.timetableItems, id: \.self) { item in
                     Button {
-                        store.send(.view(.timetableItemTapped))
+                        //store.send(.view(.timetableItemTapped))
```
Comment (shin-usu) [inline app-ios/Sources/TimetableFeature/TimetableListView.swift:94]: If you don't need it, you can delete button component.👀

Comment (charles-b-stb) [reply app-ios/Sources/TimetableFeature/TimetableListView.swift:94]: Sorry, will delete

Comment (charles-b-stb) [reply app-ios/Sources/TimetableFeature/TimetableListView.swift]: @shin-usu updated the comment to reflect better what is going on.

File: app-ios/Sources/TimetableFeature/TimetableListView.swift [hunk 2]
```diff
@@ -91,10 +91,10 @@ struct TimetableListView: View {
             LazyVStack {
                 ForEach(store.timetableItems, id: \.self) { item in
                     Button {
-                        store.send(.view(.timetableItemTapped))
+                        // Button click is in the label
```
Comment (shin-usu) [inline app-ios/Sources/TimetableFeature/TimetableListView.swift:94]: Will you add any action later?
If not, I assume following code is sufficient.
```swift
ForEach(store.timetableItems, id: \.self) { item in
    TimeGroupMiniList(contents: item, onItemTap) { store.send(.view(.timetableItemTapped($0))) }
}
```

Comment (charles-b-stb) [reply app-ios/Sources/TimetableFeature/TimetableListView.swift]: fixed

**Comment by charles-b-stb** [comment]
@shin-usu fixed as requested.

**Review by shin-usu — APPROVED** [review]
LGTM🎉🍾

**Pull request merged**